libs/components/progress-indicator/src/index.ts

```diff
@@ -3,7 +3,6 @@ export * from './lib/modules/progress-indicator/types/progress-indicator-action-
 export * from './lib/modules/progress-indicator/types/progress-indicator-action-click-progress-handler';
 export * from './lib/modules/progress-indicator/types/progress-indicator-change';
 export * from './lib/modules/progress-indicator/types/progress-indicator-display-mode-type';
-export * from './lib/modules/progress-indicator/types/progress-indicator-item-status';
```

Comment:
I verified this type is never exposed by our public API and is not referenced by any internal applications.